crates/driver/src/infra/blockchain/mod.rs

```
@@ -182,7 +182,10 @@ impl Ethereum {
             .transport()
             .execute(
                 "eth_createAccessList",
-                vec![serde_json::to_value(&tx).unwrap()],
+                vec![
+                    serde_json::to_value(&tx).unwrap(),
```

Maybe instead of `unwrap()` we can return an error (function returns `Result`)?

---

this would require a new error type and given that this unwrap never caused issues (all possible values are serializable) I prefer not to touch it.